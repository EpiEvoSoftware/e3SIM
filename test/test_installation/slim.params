cwdir:/workdir/sl984/epievosoftware/original_pipeline/test/test_installation
n_replicates:3
n_generation:3650
mut_rate:3.12e-10
within_host_reproduction:
cap_withinhost:1
seed_size:5
use_reference:
seed_host_matching_path:/workdir/sl984/epievosoftware/original_pipeline/test/test_installation/seed_host_match.csv
ref_path:/workdir/sl984/epievosoftware/original_pipeline/test/data/TB/GCF_000195955.2_ASM19595v2_genomic.fna
causal_gene_path:/workdir/sl984/epievosoftware/original_pipeline/test/test_installation/causal_gene_info.csv
use_genetic_model:T
use_network_model:T
contact_network_path:/workdir/sl984/epievosoftware/original_pipeline/test/test_installation/contact_network.adjlist
host_size:10000
epi_model:SEIR
n_epoch:1
S_IE_rate:0.004
I_R_rate:0.008
R_S_rate:0.05
latency_prob:1
E_I_rate:0.01
I_E_rate:0
E_R_rate:0
sample_rate:1e-05
recovery_prob_after_sampling:0
cap_transmissibility:10
cap_drugresist:0
transmissibility_effsize:1
drugresistance_effsize:0
n_massive_sample:0
super_infection:
