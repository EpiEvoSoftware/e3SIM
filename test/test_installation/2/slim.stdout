// Initial random seed:
1193407563351260491

// RunInitializeCallbacks():
initializeSLiMModelType(modelType = 'nonWF');
initializeSLiMOptions(nucleotideBased = T);
initializeTreeSeq();
initializeAncestralNucleotides("TTGACCGATGACCCCGGTTC...");
initializeRecombinationRate(0);
initializeMutationTypeNuc(0, 1, "f", 0);
initializeMutationTypeNuc(1, 1, "f", 0);
